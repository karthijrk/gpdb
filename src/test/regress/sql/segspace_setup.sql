--
-- Setup for segspace.sql tests
-- 

-- start_ignore
\! gpconfig -c gp_workfile_limit_per_segment -v 5242880
\! gpstop -rai
-- end_ignore
