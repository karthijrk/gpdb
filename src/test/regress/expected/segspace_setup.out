--
-- Setup for segspace.sql tests
-- 
-- start_ignore
\! gpconfig -c gp_workfile_limit_per_segment -v 5120
\! gpstop -rai
-- end_ignore
